GitHub.js

```diff
@@ -94,7 +94,13 @@ function scrape(doc, url) {
 		// long-term, but for now, let's just grab the user-facing permalink
 		let permalink = attr(doc, '.js-permalink-shortcut', 'href');
 		if (permalink) {
-			item.url = 'https://' + doc.location.host + permalink;
+			item.url = 'https://github.com' + permalink;
+		}
+		else {
+			let clipboardCopyPermalink = attr(doc, '#blob-more-options-details clipboard-copy', 'value');
```

Comment:
Seems like it could just be `'clipboard-copy'`

Reply:
There are a lot of `clipboard-copy`s on some pages - this issue discussion page has seven. The more specific selector makes sure that it only catches the URL one.

Reply:
Ah, OK